spec/models/user_spec.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: users
+#
+#  id                     :bigint           not null, primary key
+#  allow_password_change  :boolean          default(FALSE)
+#  confirmation_sent_at   :datetime
+#  confirmation_token     :string
+#  confirmed_at           :datetime
+#  email                  :string
+#  encrypted_password     :string           default(""), not null
+#  gender                 :integer
+#  image                  :string
+#  name                   :string
+#  nickname               :string
+#  provider               :string           default("email"), not null
+#  remember_created_at    :datetime
+#  reset_password_sent_at :datetime
+#  reset_password_token   :string
+#  tokens                 :json
+#  uid                    :string           default(""), not null
+#  unconfirmed_email      :string
+#  created_at             :datetime         not null
+#  updated_at             :datetime         not null
+#
+# Indexes
+#
+#  index_users_on_confirmation_token    (confirmation_token) UNIQUE
+#  index_users_on_email                 (email) UNIQUE
+#  index_users_on_reset_password_token  (reset_password_token) UNIQUE
+#  index_users_on_uid_and_provider      (uid,provider) UNIQUE
+#
+
+require 'rails_helper'
+
+describe User do
+  describe 'gender' do
+    subject { build :user }
+
+    let(:user) { create(:user, :male) }
+
+    it 'returns the correct gender' do
+      expect(user.gender).to eq('female').or eq('male')
```

Comment:
If we're mocking a male user we should test the gender is `male`. Separate this into 2 assertions, one for each gender type.
```suggestion
      let(:male_user) { create(:user, :male) }    
      let(:female_user) { create(:user, :female) }
          
      it 'returns the correct gender' do
        expect(male_user.gender).to eq('male')
        expect(female_user.gender).to eq('female')
      do
```